config/matchmaking.yml

```diff
@@ -1,4 +1,14 @@
 default: &default
+  coffee_time:
+    active: true
+    channel: rotating-coffee-time
+    schedule: daily
+    size: 2
+  q_branch:
+    active: true
+    channel: rotating-q-branch
+    schedule: daily
+    size: 2
```

Comment:
The `size` default is 2, so we could remove this to leverage that. Thoughts?

Reply:
I see what you're saying. I'm usually the slowest one in the room to pick up on something so I always appreciate when things are explicit. If it's all the same to you I'd vote to keep it in there. 🙂